config/yaml-template/yurt-tunnel-server.yaml

```diff
@@ -129,6 +129,8 @@ metadata:
   name: __project_prefix__-tunnel-server-cfg
   namespace: kube-system
 data:
+  http-proxy-ports: ""
+  https-proxy-ports: ""
```

Comment:
What about using an array rather than a comma-separated string?

Reply:
I think comma-separated string is more simpler than array string for user to edit fields in configmap resource. because array string like the following setting:
```
http-proxy-ports: '["aaa", "aaa", "aaa"]'
https-proxy-ports: '["bbb", "bbb", "bbb"]'
```
so how about keep this comma-separated string design?

Reply:
Ok, it is reasonable.